Packs/GitHub/Integrations/GitHub/GitHub.py

```diff
@@ -2037,15 +2038,17 @@ def github_trigger_workflow_command():
     repository = args.get('repository') or REPOSITORY
     branch = args.get('branch', 'master')
     workflow = args.get('workflow')
+    inputs = args.get('inputs', {})
```

Comment:
did you check if the API works with `inputs={}`?

Reply:
No. but `assign_params` will ignore it in case of an empty dict (I can leave it actually with `None`)